lib/udf_findhotel_utils.rb

```diff
@@ -152,6 +152,20 @@ def get_label_value(label, *keys):
           ~,
           tests:       [
                        ]
+      },
+      {
+          type:        :function,
+          name:        :parse_label_query_string,
```

Comment:
I would rather call it `query_string_to_json_str` or `qs_to_json_str`

Reply:
I'd agree but since it's only for "naked" qs I didn't want to cause confusion. I added a test instead!